lib/email/receiver.rb

```diff
@@ -1021,11 +1021,11 @@ def notification_level_for(body)
 
       body = body.strip.downcase
       case body
-      when "mute"
+      when "mute", "muted", I18n.t("js.topic.notifications.muted.title").downcase
```

Comment:
Will `I18n.t` here be the author locale or the site locale?

Reply:
probably site locale, but still I guess this is an improvement.